assets/js/googlesitekit/notifications/components/common/ActionsCTALinkDismiss.js

```diff
@@ -35,34 +43,51 @@ export default function ActionsCTALinkDismiss( {
 	onCTAClick,
 	onDismiss = () => {},
 	dismissLabel = __( 'OK, Got it!', 'google-site-kit' ),
+	dismissOnCTAClick = true,
 	dismissExpires = 0,
 	dismissOptions = {},
+	showCTAError = true,
 } ) {
 	const isNavigatingToCTALink = useSelect( ( select ) => {
 		return ctaLink
 			? select( CORE_LOCATION ).isNavigatingTo( ctaLink )
 			: false;
 	} );
 
+	const ctaError = useSelect( ( select ) => {
+		return select( CORE_SITE ).getError( 'notificationCTAClick', [ id ] );
+	} );
+
 	return (
```

Comment:
Note that currently on `develop`, `dismissExpires` is passed to both the `CTALink` and `Dismiss` components, but is handled differently in each of them.

Reply:
I agree - when I was refactoring this from the legacy Banner Notification components, I really wanted to keep the list of props to a minimum and hence the idea of "reusing" props. But it does make things complicated for sure. So happy with the refactor here! Thank you!